Sources/SpelledPitch/PitchSpeller/Wetherfield/Wetherfield.swift

```diff
@@ -28,8 +34,14 @@ struct PitchSpeller {
     )
 
     static let weightCarrying = WeightCarrying.build(from: PitchSpeller.tendencyGraph)
-    static let tendencyMask: WeightCarrying<WeightedGraph<Cross<Int,Tendency>,Double>> = weightCarrying.pullback { $0.b }
-
+    static let tendencyMask: WeightCarrying<WeightedGraph<FlowNode<Cross<Int,Tendency>>,Double>> = weightCarrying.pullback {
+        switch $0 {
```

Comment:
When switching over something, might as well give it a name, like `node` or something.

Reply:
`.pullback { node in switch node { ... } } ...`